lib/Graph.dart

```diff
@@ -132,26 +147,27 @@ class Node {
 
   Size size = Size(0, 0);
 
-  Offset position = Offset(0, 0);
+  Offset? position = Offset(0, 0);
```

Comment:
💡 probably this is wrong by design (but I might be wrong)

in the current implementation, the `position` - might be changed outside.  Not sure thats a good idea.
Easy fix is smth like this:
```dart
Offset? _position = Offset(0, 0);
Offset? get position => _position;
```